### PR TITLE
kata-deploy: Fix typo

### PR DESCRIPTION
**Pull request by nitkon**
Fix some typo in kata-deploy Readme

Fixes: #991

Signed-off-by: Nitesh Konkar <niteshkonkar@in.ibm.com>

### CODE REVIEW

File: kata-deploy/README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ In order to use a workload Kata with Firecracker, first add a `RuntimeClass` as:
   $ kubectl apply -f https://raw.githubusercontent.com/kata-containers/packaging/master/kata-deploy/k8s-1.13/kata-fc-runtimeClass.yaml
   ```
 
-The following YAML snippet shows how to specify a workload should use Kata with QEMU:
+The following YAML snippet shows how to specify a workload should you use Kata with QEMU:
```
Comment (grahamwhaley) [inline kata-deploy/README.md]: urgh, I know you inherited these sentences @nitkon :-), but they make me twitch (not your fault).
Let's see if we can improve them... if I take the first similar instance I found in the doc:

```
In order to use a workload Kata with QEMU
```

should probably be something simpler like:

```
To use a workload with Kata-QEMU
```

Similarly, I suspect the line you edited could be:

```
The following YAML snippet shows how to specify a workload using Kata-QEMU
```

Are you up for hunting down other instances in the doc?
/cc @jodh-intel for extra grammaticalness.....

Comment (jodh-intel) [reply kata-deploy/README.md]: Agreed, that would be great if you don't mind @nitkon ? (good to see you again by the way! :smile:)

Comment (nitkon) [reply kata-deploy/README.md]: @jodh-intel @grahamwhaley : Fixed! ;-)

File: kata-deploy/README.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ is defined through a custom resource definition. For Kubernetes 1.13:
   $ kubectl apply -f https://raw.githubusercontent.com/kata-containers/packaging/master/kata-deploy/k8s-1.13/runtimeclass-crd.yaml
 ```
 
-In order to use a workload Kata with QEMU, first add a `RuntimeClass` as:
+To use a workload with Kata-QEMU, first add a `RuntimeClass` as:
```
Comment (jodh-intel) [inline kata-deploy/README.md:96]: I may have missed previous discussion but I'm afraid I'm not a fan of this (new) term. I think "Kata for QEMU" is the clearest way to refer to this, or alternatively you could refer to "the `kata-qemu` runtime class".

Same comment for the text below.

Comment (nitkon) [reply kata-deploy/README.md]: Sure @jodh-intel . Updated.

**Pull request merged**